python/python_direct/lru_cache.h

```diff
@@ -0,0 +1,103 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <fusion.h>
+#include <runtime/fusion_executor_cache.h>
+#include <runtime/fusion_kernel_runtime.h>
+
+#include <iostream>
+#include <list>
+#include <optional>
+#include <unordered_map>
+
+class FusionKernelRuntime; // Forward declaration
```

Comment:
I don't see the usage of FusionKernelRuntime here. why do we need a forward declaration?